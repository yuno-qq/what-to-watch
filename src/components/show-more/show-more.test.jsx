import React from "react"
import {describe, test, expect} from "@jest/globals"
import renderer from "react-test-renderer"
import ShowMore from "./show-more"


describe(`<ShowMore> snapshots:`, () => {
  test(`should render button`, () => {
    const tree = renderer.create(<ShowMore onClick={() => {}} />)
      .toJSON()

    expect(tree).toMatchSnapshot()
  })
})
