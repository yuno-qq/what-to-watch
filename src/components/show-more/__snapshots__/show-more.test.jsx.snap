// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ShowMore> snapshots: should render button 1`] = `
<div
  className="catalog__more"
>
  <button
    className="catalog__button"
    onClick={[Function]}
    type="button"
  >
    Show more
  </button>
</div>
`;
